### DISCUSSION TITLE
Do I need to assign a CPU detector to a camera and is it better to reduce detection FPS upfront?

### DISCUSSION THREAD

**Discussion by WJ4IoT** [blakeblackshear/frigate#8516] [General]
I am pretty new to Frigate and awaiting a coral to use as a detector.
So in the meantime I am using CPU as a detector (and added two for two cameras).
f.y.i. my system (NUC5i5RYH) has two 2 cores 4 threads 
Is there a relation between cores and the number of CPU detectors you can create?
(According the documentation it does not seem to be).
The real question: What I think I am missing is to assign each detector to a camera or is this done automatically? 
btw the Inference Speed is roundish 108

Another question: Is it better to reduce detection FPS upfront in other words change the FPS of your (sub-)stream of your camera or will this make no difference for Frigate?

BTW I notice that you can reduce CPU (load) noticeably by using masks

**Comment by NickM-27** [comment]
Frigate handles detection automatically, you do not assign them to cameras. 

Ideally you'd lower the fps in the camera settings too, this means fewer frames get decoded by frigate and then thrown out. That said it's not a huge issue if the cameras substream fps is 15 and you're running 5 detect fps (for example)
